src/main/java/org/jpeek/App.java

```diff
@@ -200,6 +200,14 @@ public void analyze() throws IOException {
                 )
             );
         }
+        if (this.params.containsKey("LCOM4")) {
```

Comment:
@paulodamaso `analyze` is a really long method. How about we extract reports building to a separate class and use a `Map` instead of `if` statements? In that sense we can localize future changes in that class not in the main method.